public/locales/es/translation.json

```diff
@@ -15,31 +15,41 @@
       "id": "1",
       "name": "[es] Britney Johnson",
```

Comment:
Is there a reason you are putting the language code in brackets here? I have not seen that but it might be useful for knowing your current language in the future?

Reply:
Yes just putting it here because I didn't actually have translations but wanted to make sure the page actually updated! So every string in the Spanish file starts with [es] and everyone in the Chinese file starts with [zh]. I guess I don't actually need to include the language code in brackets in name strings because they probably wouldn't be translated